rexray/cli/cmds_volume.go

```diff
@@ -66,14 +66,13 @@ func (c *CLI) initVolumeCmds() {
 			}
 			if c.volumeID != "" || c.volumeName != "" {
 				for _, v := range vols {
-					if strings.ToLower(v.ID) == strings.ToLower(c.volumeID) ||
-						strings.ToLower(v.Name) == strings.ToLower(c.volumeName) {
+					if (c.volumeID != "" && strings.ToLower(v.ID) == strings.ToLower(c.volumeID)) ||
+						(c.volumeName != "" && strings.ToLower(v.Name) == strings.ToLower(c.volumeName)) {
```

Comment:
Hi @akiraw,

I'm unsure of the question you're posing here. Are you asking if the above code is the desired approach?